src/json/projects.json

```diff
@@ -0,0 +1,16 @@
+[
```

Comment:
I think we can move this file side by side with the projects overview component.

Reply:
I keep it in `json` folder in case we want to use any json in the future.
Is it ok then?

Reply:
ok